api/attestationconfigapi/fetcher.go

```diff
@@ -60,7 +74,7 @@ func newFetcherWithClientAndVerifier(client apifetcher.HTTPClient, cosignVerifie
 }
 
 // FetchLatestVersion returns the latest versions of the given type.
-func (f *fetcher) FetchLatestVersion(ctx context.Context, variant variant.Variant) (Entry, error) {
+func (f *fetcher) FetchLatestVersion(ctx context.Context, variant Variant) (Entry, error) {
```

Comment:
```suggestion
func (f *fetcher) FetchLatestVersion(ctx context.Context, variant string) (Entry, error) {
```
This might require some more changes to existing code, but why not expect `string` as input?

Reply:
because it requires changing a lot of code